lib/modules/dosomething/dosomething_helpers/dosomething_helpers.variable.inc

```diff
@@ -32,6 +32,15 @@ function dosomething_helpers_variable_form($form, &$form_state, $node) {
       'maxlength' => '140',
     ),
   ];
+  $form['custom_social_sharing']['social_share_custom_description'] = [
```

Comment:
this allows for campaign leads to put custom text above the share image like the below: 
![screen shot 2017-01-09 at 3 56 38 pm](https://cloud.githubusercontent.com/assets/9019452/21783191/46463f50-d684-11e6-821f-9b2ccec00b3f.png)